L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -527,9 +520,21 @@ void HGCalHistoSeedingImpl::findHistoSeeds(const std::vector<edm::Ptr<l1t::HGCal
     if (seedingSpace_ == RPhi) {
 
       navigator_ = Navigator(nBins1_, Navigator::AxisType::Bounded, nBins2_, Navigator::AxisType::Circular); 
-      
+     
+      for(unsigned int nb1 = 0; nb1 < nBins1_; nb1++) {
+	binsSumsHisto_[it][nb1] = binsSumsHistoN_[nb1+nbins1_counter];
```

Comment:
The counter doesn't seem to be properly initialized. And it should rather be a local variable.